src/components/AsyncTable/Placeholder.tsx

```diff
@@ -1,31 +1,38 @@
-import { TableData } from './Column';
-import Skeleton from 'react-loading-skeleton';
 import React from 'react';
+import Skeleton from 'react-loading-skeleton';
 
-interface RowPlaceholdersProps {
+import { TableData } from './Column';
+
+export interface RowPlaceholdersProps {
   amount: number;
   height: number;
   numColumns: number;
 }
 
 export const RowPlaceholders = ({ amount, height, numColumns }: RowPlaceholdersProps) => {
-  const columnPlaceholder = (
-    <TableData alignment={'center'}>
-      <Skeleton width={'100%'} height={height} />
-    </TableData>
-  );
-
-  const columns = Array(numColumns).fill(columnPlaceholder);
-  const rows = Array(amount).fill(<tr>{columns}</tr>);
+  const columns = Array(numColumns)
+    .fill(null)
+    .map((_, index) => (
+      <TableData key={index} alignment={'center'}>
```

Comment:
The reason for removing `columnPlaceholder` here was to prevent the error:

`<tr> cannot appear as a child of <div>`

I would have needed to wrap `div` tags around `columnPlaceholders` to add the key attribute and index, which would result in the above error.